cookiecutter/log.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+import logging
+import sys
+
+LOG_LEVELS = {
+    'DEBUG': logging.DEBUG,
+    'INFO': logging.INFO,
+    'WARNING': logging.WARNING,
+    'ERROR': logging.ERROR,
+    'CRITICAL': logging.CRITICAL,
+}
+
+LOG_FORMATS = {
+    'DEBUG': u'%(levelname)s [%(template)s] %(name)s: %(message)s',
+    'INFO': u'%(levelname)s: %(message)s',
+}
+
+
+class ContextFilter(logging.Filter):
+    """Logging filter that holds the name of the used template."""
+
+    def __init__(self, template):
+        super(ContextFilter, self).__init__()
+        self.template = template
+
+    def filter(self, record):
+        """Add 'template' as extra information on a logging record."""
+        record.template = self.template
+        return record
+
+
+def create_logger(template, stream_level='DEBUG', debug_file=None):
```

Comment:
Shouldn't this be named `configure_logging`? I expected something totally different given the current name (like a function that just returns a half-configured logger object).